### PR TITLE
Fix Ubuntu 22.04 package names

### PR DESCRIPTION
**Pull request by mbrukman**
The Qt 5 package names are different in Ubuntu 22.04 from prior releases (18.04 and 20.04), so we need to have a different package installation command in CI.

This is a follow-up to PR https://github.com/nuttyartist/notes/pull/365 which disabled fast-fail so we can see all Ubuntu failures independently per-version.

### CODE REVIEW

**Comment by mbrukman** [comment]
@bjorn, @nuttyartist — this PR is ready for your review, as all Ubuntu builds now pass. Probably want to remove Ubuntu 18.04 at some point as it's deprecated and will be deleted in the future, but probably OK to keep for now. Up to you.

**Comment by nuttyartist** [comment]
We should probably wait after merging @guihkx's PR #367?

**Comment by mbrukman** [comment]
Sounds like this PR isn't necessary if you're going to merge PR #367, because it changes how Qt is installed by using another GitHub Action to do it, rather than `sudo apt install` manually.

So if/when you merge that other PR, feel free to close this one, as well as my other PR #372 for macOS, since the other PR deletes both `ubuntu.yml` and `macos.yml` and merges all the builds into a single config, and also adds Windows support.

**Comment by guihkx** [comment]
Oops, I'm so sorry for stepping on your foot, I didn't realize someone else was working on this...

My PR in #367 initially used a single worflow for all three OSes and was installing Qt using a Github action, but I already have split it into three separate workflows ([linux.yml](https://github.com/guihkx/notes/actions/runs/3117196083/workflow), [mac.yml](https://github.com/guihkx/notes/actions/runs/3117196086/workflow), [windows.yml](https://github.com/guihkx/notes/actions/runs/3117196085/workflow)), and I'm also building Notes using Qt from Ubuntu's official repository too ([here](https://github.com/guihkx/notes/actions/runs/3117196083/workflow#L7-L32)).

However, for the AppImage release, I'm still using that GitHub action to install the latest official Qt release, instead of relying on the one provided by Ubuntu's repository. This is mainly because release builds are currently built using Ubuntu 18.04, just to ensure broader compatibility between distributions. But the downside is that the Qt version from Ubuntu's repository is too old (5.9.5), and I thought it would make sense to bundle the latest Qt version with the AppImage to match what I'm doing for Windows and macOS releases too.

I haven't pushed these changes yet to the branch of that PR, just to avoid spamming people's emails with my constant pushes. But as soon as I get an approval, I'll rebase and push. 

Thanks!

**Comment by nuttyartist** [comment]
> Sounds like this PR isn't necessary if you're going to merge PR #367, because it changes how Qt is installed by using another GitHub Action to do it, rather than `sudo apt install` manually.
> 
> So if/when you merge that other PR, feel free to close this one, as well as my other PR #372 for macOS, since the other PR deletes both `ubuntu.yml` and `macos.yml` and merges all the builds into a single config, and also adds Windows support.

Okay, still your contribution is very much appreciated. Thanks!

**Comment by mbrukman** [comment]
@guihkx — no worries! No offense taken and no need to apologize; your PR was actually there first, and is a much more significant contribution, whereas my changes were just simple quick fixes. Please continue doing the great job you're doing!

**Review by bjorn — APPROVED** [review]
Since this is a simple change that fixes the Ubuntu 22.04 build, I would suggest we just merge it for now.

**Comment by nuttyartist** [comment]
Semms like now we have conflicts?

**Comment by bjorn** [comment]
> Semms like now we have conflicts?

Yes, but since you merged @guihkx's PR #367 and it included the packages changes for the latest Ubuntu already, this PR is now obsolete, so we can close this one.

Thanks anyway for your contribution, @mbrukman!

**Pull request closed without merging**
